x-pack/elastic-agent/docs/elastic-agent.asciidoc

```diff
@@ -1,78 +1,255 @@
+
+==== `-path.home`
+
+Specifies home directory of an agent. `path.home` is used for determining location of configuration file or data directory.
+
+==== `-c`
+
+This flags specifies configuration file to load.
+If this is omitted elastic agent uses `{path.home}/elastic-agent.yml`.
+
+
+==== `-path.data`
+
+Data directory is used by agent to store downloaded artifacts and as a place where logs generated by Beats started and managed by the agent are written.
+
+If not specified `{path.home}/data` is used.
+
+[[elastic-agent-configuration]]
+== Configuring Elastic Agent
+
+By default Elastic Agent is configured to run in a standalone mode ingesting system data and sending them to local Elasticsearch on port 9200 with demo credentials of `elastic` user. It's also configured to monitor behavior of managed Beats in a form of tracking their logs and metrics and sending them to the same Elasticsearch instance.
+
+To alter this behavior user has following options:
+
+=== Configuring output
+
+Elastic Agent enables definition of multiple outputs where each data source can be paired with different output.
+
+E.g
+[source,yaml]
+-------------------------------------------------------------------------------------
+outputs:
+  default:
+    type: elasticsearch
+    hosts: [127.0.0.1:9200]
+    username: elastic
+    password: changeme
+
+  monitoring:
+    type: elasticsearch
+    api_key: VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw
+    hosts: ["localhost:9200"]
+    ca_sha256: "7lHLiyp4J8m9kw38SJ7SURJP4bXRZv/BNxyyXkCcE/M="
+-------------------------------------------------------------------------------------
+
+In the example above 2 outputs are provided. One called `default` and one called `monitoring`.
+Difference between them is in authentication method they are using, while first one uses username password pair second one contains api key.
+
+[NOTE]
+==============
+Configuration is invalid without definition of default output
+==============
+
+=== Enable/Disable Beats monitoring
+
+To disable or enable monitoring of managed Beats following section is available:
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+settings.monitoring:
+  # enabled turns on monitoring of running processes
+  enabled: true
+  # enables log monitoring
+  logs: true
+  # enables metrics monitoring
+  metrics: true
+  # specifies output to be used
+  use_output: monitoring
+-------------------------------------------------------------------------------------
+
+
+In case `settings.monitoring.enabled` is set to `false` monitoring of Beats is turned on and other options are ignored.
+If `settings.monitoring.enabled` is set to `true` Elastic Agent watches Metrics or Logs of Beats according to `settings.monitoring.metrics` or `settings.monitoring.logs` values.
+
+Having both set to `false` results in the same behavior as disabling monitoring.
+
+`use_output` option specifies output to which events will be send.
```

Comment:
```suggestion
Set `use_output` to specify the output to which monitoring events are sent.
```

Reply:
Is this option required, or does it use the `default` output if no other output is set?

Reply:
Oh I see later on that it is. We should mention that here.